versions/1.0.0.md

```diff
@@ -249,25 +251,29 @@ updates:
     parameters:
       - name: top
         in: query
+        # ...
       - name: skip
         in: query
+        # ...
 ```
 
-This approach allows flipping control of where Overlays apply updates to the OpenAPI description itself.
+This approach allows inversion of control as to where the Overlay updates apply to the target document itself.
 
 ### <a name="specificationExtensions"></a>Specification Extensions
 
-While the OpenAPI Specification tries to accommodate most use cases, additional data can be added to extend the specification at certain points.
+While the Overlay Specification tries to accommodate most use cases, additional data can be added to extend the specification at certain points.
 
-The extensions properties are implemented as patterned fields that are always prefixed by `"x-"`.
+The extension properties are implemented as patterned fields that are always prefixed by `"x-"`.
 
 Field Pattern | Type | Description
 ---|:---:|---
-<a name="infoExtensions"></a>^x- | Any | Allows extensions to the OpenAPI Schema. The field name MUST begin with `x-`, for example, `x-internal-id`. Field names beginning `x-oai-` and `x-oas-` are reserved for uses defined by the [OpenAPI Initiative](https://www.openapis.org/). The value can be `null`, a primitive, an array or an object.
+<a name="overlayExtensions"></a>^x- | Any | Allows extensions to the Overlay Specification. The field name MUST begin with `x-`, for example, `x-internal-id`. Field names beginning `x-oai-` and `x-oas-` are reserved for uses defined by the [OpenAPI Initiative](https://www.openapis.org/). The value MAY be `null`, a primitive, an array or an object.
```

Comment:
This actually means there's a good chance there's an error in the OAS anchor for `^x-`.

Reply:
Oh, I see what you mean - will check.